go/libraries/doltcore/sqle/dsess/session.go

```diff
@@ -86,6 +86,14 @@ func init() {
 			Type:              sql.NewSystemBoolType(TransactionsDisabledSysVar),
 			Default:           int8(0),
 		},
+		{
```

Comment:
Somewhere we should put a comment explaining what this var does, this seems like a good place